docs/src/content/docs/tutorials/split-payments.mdx

```diff
@@ -0,0 +1,296 @@
+
+```ts
+const merchantIncomingPayment = await client.incomingPayment.create(
+  {
+    url: new URL(WALLET_ADDRESS).origin,
+    accessToken: INCOMING_PAYMENT_ACCESS_TOKEN
+  },
+  {
+    merchantWalletAddress: WALLET_ADDRESS,
+    incomingAmount: {
+      value: '9900',
+      assetCode: 'USD',
+      assetScale: 2
+    },
+    expiresAt: new Date(Date.now() + 60_000 * 10).toISOString()
+  }
+)
+```
+
+</TabItem>
+
+<TabItem label = 'Platform'>
+
+```ts
+const platformIncomingPayment = await client.incomingPayment.create(
+  {
+    url: new URL(WALLET_ADDRESS).origin,
+    accessToken: INCOMING_PAYMENT_ACCESS_TOKEN
+  },
+  {
+    platformWalletAddress: WALLET_ADDRESS,
+    incomingAmount: {
+      value: '100',
+      assetCode: 'USD',
+      assetScale: 2
+    },
+    expiresAt: new Date(Date.now() + 60_000 * 10).toISOString()
+  }
+)
+```
+
+</TabItem>
+</Tabs>
+
+### 4. Request quote grants
+
+Request two quote grants from the customer wallet authorization server, as there are two recipients on this payment.
+
+```ts
+const grant = await client.grant.request(
+  {
+    url: walletAddress.authServer
+  },
+  {
+    access_token: {
+      access: [
+        {
+          type: 'quote',
+          actions: ['create', 'read']
+        }
+      ]
+    }
+  }
+)
+```
+
+### 5. Request quotes
+
+Request two quote resources from the customer wallet resource server using the access tokens returned in Step 4. Specify the respective `incomingPayment` URLs returned in Step 3 for each of the quotes.
+
+```ts
+const quote = await client.quote.create(
+  {
+    url: new URL(WALLET_ADDRESS).origin,
+    accessToken: QUOTE_ACCESS_TOKEN
+  },
+  {
+    method: 'ilp',
+    walletAddress: WALLET_ADDRESS,
+    receiver: INCOMING_PAYMENT_URL
+  }
+)
+```
+
+### 5. Request an interactive outgoing payment grant
+
+Request an `outgoingPayment` grant from the customer wallet authorization server.
+
+Add the `limits` object with the following properties:
+
+- `receiveAmount` : the total maximum amount to be received and then split between the merchant's wallet and your platform's wallet.
+
+:::note
+This request requires an interactive grant as the customer will need to consent before an `outgoingPayment` is made against their wallet.
+:::
+
+```ts
+const grant = await client.grant.request(
+  {
+    url: customerWalletAddress.authServer
+  },
+  {
+    access_token: {
+      access: [
+        {
+          identifier: merchantWalletAddress.id,
+          type: 'outgoing-payment',
+          actions: ['read', 'create'],
+          limits: {
+            receiveAmount: {
+              assetCode: receiveAmount.assetCode,
+              assetScale: receiveAmount.assetScale,
+              value: receiveAmount.value
+            }
+          }
+        }
+      ]
+    },
+    interact: {
+      start: ['redirect'],
+      finish: {
+        method: 'redirect',
+        uri: 'http://localhost:3344',
+        nonce: NONCE
+      }
+    }
+  }
+)
+```
+
```

Comment:
I don't know whether we should explain the interaction flow again, but we would need to continue the outgoing payment grant after we've been redirected back to website after the customer went through the IDP